IfcSensor Rail Import BIMQ 2021-07-21T06:58:03.5846183Z (#165)
bSI-InfraRoom/IFC-Specification: IFC4x3/Properties/d/DetectionRange_18tiQnmqr3a9LRMZ$sdhle/Documentation.md

> The range of distance that can be detected by the equipment.

Reviewer: ```suggestion
The range at which the equipment can detect changes.
```

Reviewer 2: Updated: https://github.com/bSI-InfraRoom/IFC-Specification/pull/165/commits/1e8b0740cc2e7e000d58ef2f240b7e5a49640ee9
